lib/definition/dsl/comparators.rb

```diff
@@ -36,33 +36,33 @@ def NonEmptyString # rubocop:disable Style/MethodName
       end
 
       # Example:
-      # GreaterThen(5)
-      def GreaterThen(min_value) # rubocop:disable Style/MethodName
-        Types::Lambda.new("greater_then", context: { min_value: min_value }) do |value|
+      # GreaterThan(5)
+      def GreaterThan(min_value) # rubocop:disable Style/MethodName
```

Comment:
pls add something like this to keep backwards compatibility:

```
alias GreaterThen GreaterThan
deprecate :GreaterThen
```

`deprecate` is coming with activesupport that we have included already FYI: https://apidock.com/rails/Module/deprecate